developers/wcs/embeddings/administration.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Administration
+sidebar_position: 3
+image: og/wcs/user_guides.jpg
+---
+
+import Link from '@docusaurus/Link';
+
+Weaviate Embeddings is enabled by default at the organization level and is available to all Weaviate Cloud users. 
```

Comment:
I'm a little curious as to why you chose this sentence to be the opening one for the section.